php-migrations/Emergence/People/20171014_phone-length.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+// skip conditions
+$skipped = true;
+if (!static::tableExists('people')) {
+    printf("Skipping migration because table `people` does not exist yet\n");
+    return static::STATUS_SKIPPED;
+}
+
+// migration
+if (static::getColumn('people', 'Phone')) {
```

Comment:
`getColumn` gets a bunch of details of the column, there's another call just for checking if the field exists. The skip condition here needs to be checking whether the length is already set correctly

Reply:
@themightychris I assume you're referring to `columnExists()`

Reply:
@jtwiest here's an example of a length migration: https://github.com/JarvusInnovations/emergence-skeleton/blob/master/php-migrations/Emergence/People/2014050712000_longer-password-hash.php